### ISSUE TITLE
[a11y] 11.2.4.3 Focus order

### ISSUE THREAD

**Issue by tschiebel** [owncloud/ios-app#1337]
audit: https://infinite.owncloud.com/s/qSvYucPltEYUhhc

https://github.com/BIK-BITV/BIK-App-Test/tree/main/Pr%C3%BCfschritte/de
https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf

### Personal Space

- [x] In the test, it was possible to focus non-interactive elements with the keyboard,
which should be avoided.
![image](https://github.com/owncloud/ios-app/assets/132444655/edad55f6-9fa8-44e4-bf04-432f61d29ae9)

> **Fix:** The summary text can no longer be focused with the keyboard.

### Shared with me

- [x] In the test, it was possible to focus the heading "Shared with me" with the keyboard, which should be avoided.
which should be avoided.

> No longer reproduceable as of 12.2.

### Shared with others

- [x] In the test, it was possible to focus the heading "Shared with others" with the keyboard, which should be avoided.
which should be avoided.

> No longer reproduceable as of 12.2.

### Spaces

- [x] In the test, it was possible to focus on the "Spaces" heading using the keyboard, which
should be avoided.

> No longer reproduceable as of 12.2.

### Add Account

- [x] In the test, it was possible to focus the "Add account" heading with the keyboard, which should be avoided.
which should be avoided.

> No longer reproduceable as of 12.2.

### Image Viewer

- [x] In the test, it was possible to focus the heading with the file name using the keyboard.
which should be avoided.

> No longer reproduceable as of 12.2. If the filename can be focused, it now is because it provides access to a popup menu that allows changing to the parent folders of the presented file.

**Comment by jesmrec** [comment]
fixed
